src/utils/url-paths.ts

```diff
@@ -9,6 +8,10 @@ import { join } from "path";
  */
 export const absolutePathRegex = /^(?:[a-z]+:)?\/\//;
 
+const fixSlash = (path: string) => {
```

Comment:
I am a little worried about this change, but it should probably be fine. So long as everything still builds and works, it's probably not causing any problems